rafthttp/util.go

```diff
@@ -175,3 +175,14 @@ func setPeerURLsHeader(req *http.Request, urls types.URLs) {
 	}
 	req.Header.Set("X-PeerURLs", strings.Join(peerURLs, ","))
 }
+
+// addRemoteFromRequest add remote according to request header
+func addRemoteFromRequest(tr Transporter, r *http.Request) bool {
```

Comment:
just `func addRemoteFromRequest(tr Transporter, r *http.Request) {` since nothing uses the return value